docs/RELEASE_NOTES.md

```diff
@@ -27,6 +27,15 @@
 7. Add Migration guide section which specifies explicitly if there are breaking changes and how to tackle them.
 -->
 
+### 5.1.1 - May 7, 2021
+[Commits](https://github.com/commercetools/commercetools-sync-java/compare/5.1.0...5.1.1) |
+[Javadoc](https://commercetools.github.io/commercetools-sync-java/v/5.1.1/) |
+[Jar](https://bintray.com/commercetools/maven/commercetools-sync-java/5.1.1)
+
+- 🐞 **Bug Fixes** (1)
+    - **Product Sync** - Fixed a bug in the `ProductSync` related to querying of productProjections based on key values, when key value contains special characters(") [#269](https://github.com/commercetools/commercetools-project-sync/issues/269)
```

Comment:
```suggestion
    - **Product Sync** - Special characters can be defined for ProductDraft key. [#269](https://github.com/commercetools/commercetools-project-sync/issues/269)
```

Reply:
It was always allowed to define special characters in the ProductDraft key. We didn't change this. We fixed the code where we are building query for the search based on Key's.
> Special characters can be defined for ProductDraft key.
May be this will mis guide?

Reply:
ok, in that case, please make sure the mention the end result for the library user. If you mention the internal bug fixes, in the release note, you should always describe those fixes who the end-user can be benefited.

Reply:
> please make sure to mention the end result for the library user
Then your statement makes sense for me. It explains users to have special characters in the productDraft key.